c2corg_ui/static/js/constants.js

```diff
@@ -31,7 +31,8 @@ app.constants = {
   REQUIRED_FIELDS : {
     waypoints: ['title' , 'lang', 'waypoint_type', 'elevation', 'longitude', 'latitude'],
     routes : ['title' , 'lang', 'activities', 'waypoints'],
-    outings : ['title' , 'lang', 'date_start', 'routes', 'activities']
+    outings : ['title' , 'lang', 'date_start', 'routes', 'activities'],
+    images: ['title', 'image_type']
```

Comment:
I think that would be more relevant that ``title`` is not a mandatory attribute for images (as on Facebook for instance). In v5 many people uses dummy strings as image titles => that would be more relevant to keep the title blank...